developer/deploy-ipaserver.yml

```
@@ -0,0 +1,39 @@
+---
+- name: Deploy IPA server for WebUI development
+  hosts: ipaserver
+  gather_facts: true
+  become: true
+
+  roles:
+    - name: Deploy IPA server
+      role: freeipa.ansible_freeipa.ipaserver
+      state: present
+
+- name: Configure modern-ui alias in httpd
+  hosts: ipaserver
+  gather_facts: false
+  become: true
+  tasks:
+    - name: Ensure webui is configured in ipa.conf
+      ansible.builtin.blockinfile:
+        path: /etc/httpd/conf.d/ipa.conf
+        insertafter: "EOF"
+        marker: "# {mark} Webui Development configuration"
+        prepend_newline: true
+        block: |+
+          Alias /ipa/modern-ui "/webui/dist"
+          <Directory "/webui/dist">
+            SetHandler None
+            AllowOverride None
+            Satisfy Any
+            Require all granted
+            RewriteEngine On
+            RewriteRule ^(.*)/assets/(.*)$ assets/\\$2 [L]
```

```suggestion
            RewriteRule ^(.*)/assets/(.*)$ assets/$2 [L]
```

The `\\` was only added because in ipa we use jinja templates and I had to escape, the result actually looks as suggested.

---

fixed.